packages/adp-tooling/package.json

```diff
@@ -14,9 +14,10 @@
     "author": "@SAP/ux-tools-team",
     "main": "dist/index.js",
     "scripts": {
-        "build": "pnpm build:server && pnpm build:client",
+        "build": "pnpm build:server && pnpm build:client && pnpm copy:templates",
         "build:server": "tsc --build",
         "build:client": "tsc --noEmit --project src/preview/client/tsconfig.json && babel --extensions .ts ./src/preview/client -d ./dist/preview/client",
+        "copy:templates": "rimraf ./dist/templates && shx cp -r ./templates/rta ./dist/templates",
```

Comment:
We don't copy the templates into `dist`. The `templates` folder is part of the delivered package, so you can reference it there. You can check the coding in `writer/index.ts`

Reply:
Done in https://github.com/SAP/open-ux-tools/pull/1183/commits/48a9f41847b6978b602d2596b21102659e136323